_notices/rsn0017.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 17 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Deprecation announcement for CUDA 11.0 in v22.06"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v22.06+"
+notice_created: 2022-06-08
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-06-08
+---
+
+## Overview
+
+`CUDA 11.0` will be deprecated in our upcoming release of `v22.06`
+scheduled for June 2022. After the `v22.06` release, users should expect 
+that support for `CUDA 11.0` may be dropped at any point.
+
+## Impact
+
+Users should plan to move to `CUDA 11.2+` as soon as possible. Future
```

Comment:
We should mention CUDA enhanced compatibility here as well. Believe required driver is r450.80.02+, but please confirm